phasehq/kubernetes-secrets-operator
# Fix: Local Path Secret Referencing

## Description

This PR addresses a bug in our Kubernetes operator related to secret referencing. The fix ensures that the operator correctly handles and resolves secret references within Kubernetes resources.

## Changes

- Modified the secret resolution logic in the operator to correctly handle local path references.

To test this change:
1. Deploy the updated operator to a test Kubernetes cluster.
2. Create a local secret reference across a path in Phase (e.g., `${/path/to/SECRET}`).
3. Deploy that application via the Phase Kubernetes custom CR.
4. Fetch the secret via kubctl and base64 decode it and verify the secret reference.
